index.html

```diff
@@ -1566,6 +1578,23 @@ <h3>Payload Sizes</h3>
       <p>Larger transactions can trigger DoS incidents. It's recommended to configure the payload size accepted by endpoints at an instance level.</p>
     </section>
 
+    <section>
+      <h3>Additional Validation</h3>
+      <p>
+        In most cases, simply verifying the proof might not be sufficient to properly handle the recieved data.
+        Verifier services are expected to configure additional validation steps based on their use cases. To define such
+        additional validations, implementers can refer to specifications such as the [[VC-DATA-INTEGRITY]] 
```

Comment:
We should refer to specific sections in specifications by using markup like this:

```
<a data-cite="VC-DATA-INTEGRITY#foobar">Section 3.4: Foobar</a> in the [[[VC-DATA-INTEGRITY]]] specification.
```